frontend/src/metabase/search/components/SearchResult.unit.spec.js

```diff
@@ -23,6 +29,55 @@ function collection({
   return collection;
 }
 
+const setupTableSearchResult = tableOpts => {
```

Comment:
Looks great, but nit: that we use `setup` for the function name and `SetupOpts` for the type. But in this case, it's a js test file you could–we almost always as well–destructure the prop.

```suggestion
const setup = ({ tableOpts }) => {
```

Reply:
Fixed in [3163f7e](https://github.com/metabase/metabase/pull/31651/commits/3163f7e3e83574173d775b32b2fdbd191958454f)